tests/test_utils.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+
+from app import create_app, db
+from app.models import User, BucketList, BucketListItem
+
+
+class BaseTest(unittest.TestCase):
+    """
+    Base test to be used by each test method.
+    The test details are loaded each time a test is being run.
+    """
+
+    def setUp(self):
+        self.app = create_app('testing')
+        self.client = self.app.test_client()
+        self.app_context = self.app.app_context()
+        self.app_context.push()
+
+        db.create_all()
+
+        self.create_test_user()
+        self.user = User.query.filter_by(username='damidee').first()
```

Comment:
instead of creating a User then querying the db for the user you have just created why not return said user after creating them?